app/src/main/java/com/devshish/internship/Modules.kt

```diff
@@ -1,27 +1,58 @@
 package com.devshish.internship
 
-import com.devshish.internship.data.repository.AlbumsRepository
-import com.devshish.internship.data.repository.ProfileRepository
-import com.devshish.internship.data.repository.SongsRepository
+import com.devshish.internship.MusicLibrary.*
+import com.devshish.internship.data.repository.*
 import com.devshish.internship.domain.repository.IAlbumsRepository
 import com.devshish.internship.domain.repository.IProfileRepository
-import com.devshish.internship.domain.repository.LikedSongsRepository
-import com.devshish.internship.presentation.ui.likedalbums.AlbumsViewModel
-import com.devshish.internship.presentation.ui.likedsongs.LikedSongsViewModel
+import com.devshish.internship.domain.repository.ISongsRepository
+import com.devshish.internship.presentation.ui.albums.details.AlbumDetailsViewModel
+import com.devshish.internship.presentation.ui.albums.liked.LikedAlbumsViewModel
+import com.devshish.internship.presentation.ui.albums.local.LocalAlbumsViewModel
+import com.devshish.internship.presentation.ui.songs.liked.LikedSongsViewModel
+import com.devshish.internship.presentation.ui.songs.local.LocalSongsViewModel
 import com.devshish.internship.presentation.ui.profile.EditProfileViewModel
 import com.devshish.internship.presentation.ui.profile.ProfileViewModel
 import org.koin.androidx.viewmodel.dsl.viewModel
+import org.koin.core.qualifier.named
 import org.koin.dsl.module
 
+enum class MusicLibrary {
+    SONGS_LIKED,
+    SONGS_LOCAL,
+    ALBUMS_LIKED,
+    ALBUMS_LOCAL,
+    ALBUM_SONGS
+}
```

Comment:
Enum not for ALL implementations, but for each type 
For example: enum for songs, enum for albums and so on